test/reline/yamatanooroti/test_rendering.rb

```diff
@@ -52,6 +52,14 @@ def teardown
       ENV.delete('RELINE_TEST_PROMPT') if ENV['RELINE_TEST_PROMPT']
     end
 
+    # Yamatanooroti::TestCase#write converts 0x80-0xff bytes to "\e"+(byte&0x7f).chr
+    # This method avoids the conversion and writes the string as is.
+    def write_without_meta_conversion(str)
+      sync
+      @pty_input.write(str)
+      try_sync
+    end
```

Comment:
Ideally, I think changing yamatanooroti's `write` to not do this conversion is better.
But if we do that, there will be no way to input meta key to yamatanooroti-windows and yamatanooroti-vterm in a common way.